### PR TITLE
Add proxy cache ui

### PR DESCRIPTION
**Pull request by AllForNothing**
Temp Env: https://10.185.12.35
UX: https://www.figma.com/proto/cPY9Lh8LbIQawAJSGmi1KA/Harbor-v2.0?node-id=1%3A2481&scaling=min-zoom
Changed points:
image
image
image

Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/project/create-project/create-project.component.html [hunk 1]
```diff
@@ -58,6 +58,45 @@ <h3 class="modal-title">{{'PROJECT.NEW_PROJECT' | translate}}</h3>
...
+                        <span class="alert-label">{{"REPLICATION.NO_ENDPOINT_INFO" | translate}}</span>
+                        <a class="alert-label go-link" routerLink="/harbor/registries">{{'REPLICATION.ENDPOINTS' | translate}}</a>
+                    </div>
+                </div>
+            </div>
+            <div class="clr-form-control mt-05" *ngIf="isSystemAdmin && enableProxyCache">
+                <label for="create_project_storage_limit" class="clr-control-label"></label>
+                <div class="clr-control-container">
+                    <div class="clr-input-wrapper">
+                        <label class="clr-control-label endpoint">{{ 'PROJECT.ENDPOINT' | translate }}</label>
```
Comment (jwangyangls) [inline src/portal/src/app/project/create-project/create-project.component.html:94]: image

Comment (jwangyangls) [reply src/portal/src/app/project/create-project/create-project.component.html:94]: image

File: src/portal/src/app/project/project-detail/project-detail.component.html [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 
-<h1 class="custom-h2" sub-header-title>{{currentProject.name}} <span class="role-label"
-        *ngIf="isMember">{{roleName | translate}}</span></h1>
+<h1 class="custom-h2" sub-header-title>
+    <clr-icon *ngIf="isProxyCacheProject" shape="cloud-traffic" size="36"></clr-icon>
+    <span class="ml-05">{{currentProject.name}}</span>
+    <span class="ml-05 role-label" *ngIf="isMember">{{roleName | translate}}</span>
+</h1>
+<div class="clr-row mt-0 line-height-10" *ngIf="isProxyCacheProject">
+    <span class="proxy-cache">{{ 'PROJECT.PROXY_CACHE' | translate }}</span>
```
Comment (jwangyangls) [inline src/portal/src/app/project/project-detail/project-detail.component.html:10]: image
image

File: src/portal/src/app/project/project-detail/project-detail.component.html [hunk 3]
```diff
@@ -1,8 +1,14 @@
 <a *ngIf="hasSignedIn" (click)="backToProject()" class="backStyle"> {{'PROJECT_DETAIL.PROJECTS' | translate}}</a>
 <a *ngIf="!hasSignedIn" [routerLink]="['/harbor', 'sign-in']"> {{'SEARCH.BACK' | translate}}</a>
 
-<h1 class="custom-h2" sub-header-title>{{currentProject.name}} <span class="role-label"
-        *ngIf="isMember">{{roleName | translate}}</span></h1>
+<h1 class="custom-h2" sub-header-title>
+    <clr-icon *ngIf="isProxyCacheProject" shape="cloud-traffic" size="36"></clr-icon>
```
Comment (jwangyangls) [inline src/portal/src/app/project/project-detail/project-detail.component.html:5]: The picture is a bit big compared to the design

File: src/portal/src/app/project/create-project/create-project.component.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import { debounceTime, distinctUntilChanged, filter, map, switchMap } from 'rxjs/operators';
+import { debounceTime, distinctUntilChanged, filter, finalize, map, switchMap } from 'rxjs/operators';
```
Comment (selina-cui) [inline src/portal/src/app/project/create-project/create-project.component.ts:14]: Is finalize used?

Comment (AllForNothing) [reply src/portal/src/app/project/create-project/create-project.component.ts:14]: Removed

File: src/portal/src/app/project/project-detail/project-detail.component.html (hunk 3 above)
Comment (AllForNothing) [reply src/portal/src/app/project/project-detail/project-detail.component.html:5]: fixed

File: src/portal/src/app/project/project-detail/project-detail.component.html (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/app/project/project-detail/project-detail.component.html:10]: fixed

File: src/portal/src/app/project/create-project/create-project.component.html (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/project/create-project/create-project.component.html:94]: should be consistent  with  the left labels

**Pull request merged**